scripts/lib.sh

```diff
@@ -65,14 +65,13 @@ versionNumber() {
 
 test_req_yq() {
   if ! command -v yq &>/dev/null; then
-    err 'yq and jq is required to install Dynatrace function. Please refer to following links for installation instructions:
+    err 'yq (4.9.x) and jq is required to install Dynatrace function. Please refer to following links for installation instructions:
```

Comment:
Don't want to stall you with this PR so I'll approve. Just wanted to mention that here we're saying 4.9.x. In the condition in line 85 we're checking for >=4.9.8. In the error message in line 86 we're saying 4+. It's all a bit chaotic. Please unify this if you wish, but no pressure from my side

Reply:
Fully agree that became chaotic. Scripts require yq in version 4.9.8 or higher.